src/Simulator/IncrementalRedstoneSimulator.cpp

```diff
@@ -503,152 +514,280 @@ void cIncrementalRedstoneSimulator::HandleRedstoneButton(int a_RelBlockX, int a_
 
 void cIncrementalRedstoneSimulator::HandleRedstoneWire(int a_RelBlockX, int a_RelBlockY, int a_RelBlockZ)
```

Comment:
But I have already, with the FindAndPowerBorderingWires and PowerBorderingWires, no?